DAOS-16257 ucx: Test later MOFED ucx version.

jgmoore-or opened the pull request:
Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-func-test-vm: true
Skip-unit-tests: true
Skip-nlt: true
Test-tag: pr daily_regression
Test-provider: ucx+ud_x
Skip-func-hw-test-medium-ucx-provider: false
Skip-func-hw-test-medium-verbs-provider: true

Required-githooks: true

### Before requesting gatekeeper:

* [ ] Two review approvals and any prior change requests have been resolved.
* [ ] Testing is complete and all tests passed or there is a reason documented in the PR why it should be force landed and forced-landing tag is set.
* [ ] `Features:` (or `Test-tag*`) commit pragma was used or there is a reason documented that there are no appropriate tags for this PR.
* [ ] Commit messages follows the guidelines outlined [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments).
* [ ] Any tests skipped by the ticket being addressed have been run and passed in the PR.

### Gatekeeper:

* [ ] You are the appropriate gatekeeper to be landing the patch.
* [ ] The PR has 2 reviews by people familiar with the code, including appropriate owners.
* [ ] Githooks were used. If not, request that user install them and check copyright dates.
* [ ] Checkpatch issues are resolved.  Pay particular attention to ones that will show up on future PRs.
* [ ] All builds have passed.  Check non-required builds for any new compiler warnings.
* [ ] Sufficient testing is done. Check feature pragmas and test tags and that tests skipped for the ticket are run and now pass with the changes.
* [ ] If applicable, the PR has addressed any potential version compatibility issues.
* [ ] Check the target branch.   If it is master branch, should the PR go to a feature branch?  If it is a release branch, does it have merge approval in the JIRA ticket.
* [ ] Extra checks if forced landing is requested
  * [ ] Review comments are sufficiently resolved, particularly by prior reviewers that requested changes.
  * [ ] No new NLT or valgrind warnings.  Check the classic view.
  * [ ] Quick-build or Quick-functional is not used.
* [ ] Fix the commit message upon landing. Check the standard [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments). Edit it to create a single commit. If necessary, ask submitter for a new summary.

jgmoore-or committed:
DAOS-16257 ucx: Test later MOFED ucx version.

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-func-test-vm: true
Skip-unit-tests: true
Skip-nlt: true
Test-tag: pr daily_regression
Test-provider: ucx+ud_x
Skip-func-hw-test-medium-ucx-provider: false
Skip-func-hw-test-medium-verbs-provider: true

Required-githooks: true

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 commented:
Test stage Test RPMs on EL 8.6 completed with status FAILURE.  https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-15505/1/display/redirect

jgmoore-or committed:
DAOS-16257 ucx: Update CI to use latest UCX (experimental).

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-func-test-vm: true
Skip-unit-tests: true
Skip-nlt: true
Test-tag: pr daily_regression
Test-provider: ucx+ud_x
Skip-func-hw-test-medium-ucx-provider: false
Skip-func-hw-test-medium-verbs-provider: true

Required-githooks: true

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 commented:
Test stage Test RPMs on EL 8.6 completed with status FAILURE.  https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-15505/2/display/redirect

jgmoore-or committed:
DAOS-16257 ucx: Update UCX version in CI.

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-func-test-vm: true
Skip-unit-tests: true
Skip-nlt: true
Test-tag: pr daily_regression
Test-provider: ucx+ud_x
Skip-func-hw-test-medium-ucx-provider: false
Skip-func-hw-test-medium-verbs-provider: true

Required-githooks: true

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-15505/3/execution/node/831/log

daosbuild1 commented:
Test stage Functional Hardware Medium completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-15505/3/execution/node/975/log

jgmoore-or committed:
DAOS-16257 ucx: Test with latest MOFED.

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-func-test-vm: true
Skip-unit-tests: true
Skip-nlt: true
Test-tag: pr daily_regression
Test-provider: ucx+ud_x
Skip-func-hw-test-medium-ucx-provider: false
Skip-func-hw-test-medium-verbs-provider: true

Required-githooks: true

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

jgmoore-or committed:
DAOS-16527 ucx: Test with latest MOFED.

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-func-test-vm: true
Skip-unit-tests: true
Skip-nlt: true
Test-tag: pr daily_regression
Test-provider: ucx+ud_x
Skip-func-hw-test-medium-ucx-provider: false
Skip-func-hw-test-medium-verbs-provider: true

Required-githooks: true

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-15505/5/execution/node/878/log

jgmoore-or committed:
DAOS-16257 ucx: Test with UCX shipping with latest MOFED.

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-func-test-vm: true
Skip-unit-tests: true
Skip-nlt: true
Test-tag: pr daily_regression
Test-provider: ucx+ud_x
Skip-func-hw-test-medium-ucx-provider: false
Skip-func-hw-test-medium-verbs-provider: true

Required-githooks: true

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

jgmoore-or committed:
DAOS-16257 ucx: Try UCX with latest MOFED.

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-func-test-vm: true
Skip-unit-tests: true
Skip-nlt: true
Test-tag: pr daily_regression
Test-provider: ucx+ud_x
Skip-func-hw-test-medium-ucx-provider: false
Skip-func-hw-test-medium-verbs-provider: true

Required-githooks: true

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

brianjmurrell commented on ci/provisioning/post_provision_config.sh:
Apologies.  My mistake.  Change this to:
```suggestion
echo "MLNX_VER_NUM=${MLNX_VER_NUM:-}"
```

To avoid any unbound variable errors.
